quickstarts/jumpstart/config.yml

```diff
@@ -22,9 +22,11 @@ authors:
 
 # Keywords for filtering / searching criteria in the UI
 keywords:
-  - Jumpstart
-  - Quickstart
-  - Intro
+  - jumpstart
+  - quickstart
+  - intro
+  - how-to
```

Comment:
```suggestion
```

Reply:
this feels unnecessary as a keyword so I suggest we remove it.

Reply:
Sure, the main thing was to fix the case since the querying is case sensitive.